### PR TITLE
Scan list changes

### PR DESCRIPTION
**Pull request by AmartC**
Will return a dictionary of all necessary information pertaining to scanners configured in the system.

### CODE REVIEW

File: Atomic/scan.py [hunk 1]
```diff
@@ -28,6 +28,22 @@ def __init__(self):
         self.rootfs_mappings = {}
         self.scanner = None
 
+    def get_scanners_list(self):
+        scan_dict = dict()
+        if len(self.scanners) == 0:
+            scan_dict['error'] = dict()
```
Comment (rhatdan) [inline Atomic/scan.py]: Return empty list

**Comment by rhatdan** [comment]
Can you squash your commits together?

**Comment by rhatdan** [comment]
Should these be returning a json file?

**Comment by rhatdan** [comment]
@AmartC Please rebase.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Comment by rhatdan** [comment]
@jlebon @cgwalters Homu complaining about pip?

**Comment by jlebon** [comment]
The Dockerfile.system container is failing to build because:

```
Step 1 : FROM fedora
 ---> cd1e9127a3f8
Step 2 : RUN dnf install -y nmap-ncat && dnf clean all
 ---> Running in 257d4b373d2b
Error: Failed to synchronize cache for repo 'updates'
```

@rh-atomic-bot retry

**Comment by cgwalters** [comment]
It's already merged so we can't do a retry now.

**Pull request merged**